sdk/src/main/java/com/bugsnag/android/DefaultDelivery.java

```diff
@@ -14,6 +14,7 @@
 
 class DefaultDelivery implements Delivery {
 
+    private static final int HTTP_BAD_REQUEST = 400;
```

Comment:
It seems misleading to me to use the status code `400`, which implies that a server responded in a certain way when we don't know what happened at all.

If `null` or something that denotes an absence of a value is not ok to return from this function, I'd suggesting using `0`?

Reply:
Updated to use `0` for the status code.